modules/noise/doc_classes/FastNoiseLite.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FastNoiseLite" inherits="Noise" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		Generates noise using the FastNoiseLite library.
+	</brief_description>
+	<description>
+		This class generates noise using the FastNoiseLite library, which is a collection of several noise algorithms including Cellular, Perlin, Value, and more.
+		Most generated noise values are in the range of [code][-1,1][/code], however not always. Some of the cellular noise algorithms return results above [code]1[/code].
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="cellular_distance_function" type="int" setter="set_cellular_distance_function" getter="get_cellular_distance_function" enum="FastNoiseLite.CellularDistanceFunction" default="0">
+			Determines how the distance to the nearest/second-nearest point is computed. See [enum CellularDistanceFunction] for options.
+		</member>
+		<member name="cellular_jitter" type="float" setter="set_cellular_jitter" getter="get_cellular_jitter" default="0.45">
+			Maximum distance a point can move off of its grid position. Set to [code]0[/code] for an even grid.
+		</member>
+		<member name="cellular_return_type" type="int" setter="set_cellular_return_type" getter="get_cellular_return_type" enum="FastNoiseLite.CellularReturnType" default="0">
+			Return type from cellular noise calculations. See [enum CellularReturnType].
+		</member>
+		<member name="color_ramp" type="Gradient" setter="set_color_ramp" getter="get_color_ramp">
+			A [Gradient] which is used to map the luminance of each pixel to a color value.
```

Comment:
Why isn't it just called `gradient`?

Reply:
I dont know if that's related since I didnt implement this PR, but when speaking of noise, "gradient" can also relate to the derivative of noise (which in practical terms gives normals, or displacement vectors instead of a single value). This property is for the very specific use case of generating a colored image with a fixed conversion formula used exclusively in the `get_image` function. Maybe it could be called `color_gradient`?

Reply:
Ramp is the term used in industry standard VFX tools like Adobe AfterFX and Houdini, even Blender. 

Gradient is a type of noise. 
https://en.m.wikipedia.org/wiki/Gradient_noise 

Godot is the oddball by calling color ramps gradients.

Reply:
Interesting. Godot 2 used the name ColorRamp, but in Godot 3 it was renamed to Gradient. If it goes against the rest of the industry then that seems like a strange change, and I can see how it's confusing in the context of Gradient noise, but without this context it seems like a good rename (Gradient sounds better to me).

Reply:
As Zylann suggested, I think `color_gradient` (or in this specific case `modulation_color_gradient`) would be fine, since it sets it apart enaugh from a gradient in the context of noise generation.